less/navigation.less

```diff
@@ -1 +1,58 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+.navigation {
+    display: flex;
+    width: 100%;
+    border-bottom: 2px dashed @navigation-border;
+    justify-content: space-evenly;
+    padding: 2%;
+
+    nav {
+        display: flex;
+        font-size: 1.5rem;
+        width: 800px;
+        padding-right: 20%;
+
+        a {
+            display: flex;
+            justify-content: center;
+            width: 100%;
+            padding: 1%;
+            text-decoration: none;
+            align-items: center;
+            color: black;
+        }
+    }
+
+    h1 {
+        display: flex;
```

Comment:
If you add the class container along with navigation in the div with class navigation then you won't need the padding-left here.  The width of 100 percent is still needed.